packages/web-runtime/src/services/auth/publicLinkManager.ts

```diff
@@ -40,6 +40,14 @@ export class PublicLinkManager {
     sessionStorage.setItem(PublicLinkManager.buildStorageKey(token, 'resolved'), resolved + '')
   }
 
+  setType(token: string, type: 'ocm' | 'public-link'): void {
```

Comment:
Saw `'ocm' | 'public-link'` quite a few times now... maybe makes sense to introduce a `LinkShareType` type or something similar...